src/client.js

```diff
@@ -364,6 +367,7 @@ export function MatrixClient(opts) {
     this._cryptoCallbacks = opts.cryptoCallbacks || {};
 
     this._forceTURN = opts.forceTURN || false;
+    this._supportsTransfer = opts.supportsTransfer || false;
```

Comment:
Just "transfer" seems too generic at this level... Some day it might have a meaning outside of calls, for example.

```suggestion
    this._supportsCallTransfer = opts.supportsCallTransfer || false;
```